### PR TITLE
chore: changelog for patch release v1.3.2

### PR DESCRIPTION
**Pull request by gohil-jay**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 ### Version 1.3.2
 
+* Added `storage_type` operator and `storage_type()` function [#781][], with pending deprecation for _storage_type [#786][] [#790][]
```
Comment (henryiii) [inline docs/changelog.md:15]:
```suggestion
* Added `storage_type` operator and `storage_type()` function [#781][], with pending deprecation for `_storage_type` [#786][] [#790][]
```

File: docs/changelog.md [hunk 2]
```diff
@@ -12,6 +12,10 @@
...
+* Better errors generated for missing or incorrect sample to mean storage [#782][]
+* Explicit reset() documentation [#783][]
+* Added Python 3.11 test wheels [#789][]
```
Comment (henryiii) [inline docs/changelog.md:18]:
```suggestion
* Added Python 3.11 wheels [#789][]
```

File: docs/changelog.md [hunk 3]
```diff
@@ -12,6 +12,10 @@
 
 ### Version 1.3.2
 
+* Added `storage_type` operator and `storage_type()` function [#781][], with pending deprecation for _storage_type [#786][] [#790][]
+* Better errors generated for missing or incorrect sample to mean storage [#782][]
+* Explicit reset() documentation [#783][]
```
Comment (henryiii) [inline docs/changelog.md:17]:
```suggestion
* Explicit `reset()` documentation [#783][]
```

Comment (henryiii) [reply docs/changelog.md:17]: Though I'm okay to not mention docs improvements, at least specifically. It's fine though. :)

**Comment by henryiii** [comment]
I need to make the needs changelog bot smarter! This literally _is_ a changelog update. I think I set it up better on pybind11 (though there it's always after the PR, but maybe we should do that too since we aren't adding these as we go very well as it is).

File: docs/changelog.md (hunk 3 above)
Comment (gohil-jay) [reply docs/changelog.md]: Yeah I thought about that too, but then I thought we can add it to the issue to close it completely. I just realized we already closed it before with a previous PR merge (https://github.com/scikit-hep/boost-histogram/issues/567)

**Pull request merged**